sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/transform/BeamSetOperatorsTransforms.java

```diff
@@ -17,6 +17,7 @@
  */
 package org.apache.beam.sdk.extensions.sql.impl.transform;
 
+import com.google.common.collect.Iterators;
```

Comment:
If I didn't miss anything we should use vendored guava: https://lists.apache.org/thread.html/6cdeb1f7cf6037c84a249268f641fa1caa52a721af017a1b643d60fa@%3Cdev.beam.apache.org%3E

Reply:
Yea, we depend on non-Vendored Guava as well, because `ValuesRel` requires an `ImmutableList` to a protected constructor. Other place we should not use it.

Reply:
using vendored Guava now.